styles/blocks/02-heading.json

```diff
@@ -0,0 +1,13 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/theme.json",
+	"version": 3,
+	"title": "Heading",
+	"slug": "text-heading",
+	"blockTypes": ["core/heading", "core/paragraph"],
```

Comment:
Same here, absolutely no to a paragraph being styled as a heading.

Reply:
This would lead to users creating inaccessible content.

Reply:
> Same here, absolutely no to a paragraph being styled as a heading.

I think you're missing the point. This is a text _style_. You can set text to whatever style, size, format you'd like—but the theme provides you defaults to pick from. Patterns are already doing this today, in this theme—they're just applied one off combinations of font size and line height. I am attempting to reduce the one-offs for a consistent systematized approach here.

Reply:
Also, I changed the name. I did ask for feedback on what to name this one as well.